scripts/microbe_stage/microbe.as

```diff
@@ -504,6 +504,8 @@ class MicrobeSystem : ScriptSystem{
         applyCellMovement(components, logicTime);
 
         compoundAbsorberComponent.setAbsorbtionCapacity(microbeComponent.capacity);
+        
+        membraneComponent.setHealthPercentage(microbeComponent.hitpoints / microbeComponent.maxHitpoints);
```

Comment:
There should be a check here to see if the value changed in order to not update the material of *all* cells on each frame, which I think this version does.